README.md

```diff
@@ -2,72 +2,96 @@
 
 # Project Name
 
-> Description the project.
+> Capstone Project:
 
+In this project I added the following changes:
 
-## Built With
+Milestone 0 - project setup (0.5h)
+- Set up the repository and tools.
 
-- Major languages
-- Frameworks
-- Technologies used
+Milestone 1 - content (1h)
+- Choose the topic for your website.
+- Choose images, text, icons, fonts.
 
-## Live Demo (if available)
+Milestone 2 - mobile first (5h)
+- Create the 2 pages for mobile.
+- Deploy the project.
 
-[Live Demo Link](https://livedemo.com)
+Milestone 3 - desktop version (5h)
+- Adapt the 2 pages to desktop.
 
+Milestone 4 - interactions (1.5h)
+- Implement the user interactions: link, mobile menu.
+- Deploy the project.
 
-## Getting Started
+Milestone 5 - dynamic page (5h)
+- Implement the section "Featured speakers" with dynamic HTML.
+- Deploy the project and check to make sure everything works as planned.
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+Milestone 6 - documentation (1.5h)
```

Comment:
- [ ] Instead of writing the project requirements in this section of your README file, I'll suggest you write a description of your website. for example what it talks about,...